crates/harp/src/vector/list.rs

```diff
@@ -75,8 +78,8 @@ impl super::Vector for List {
         let mut data = data.into_iter();
 
         let size = data.len();
-        let inner = crate::alloc_list(size).unwrap();
-        let inner: RObject = inner.into();
+        let sexp = crate::alloc_list(size).unwrap();
```

Comment:
Could reword this a little to use `sexp` directly in the loop and rename `inner` to `obj` here